virtme/commands/run.py

```diff
@@ -356,7 +366,9 @@ def do_script(shellcmd, use_exec=False):
     if args.script_exec is not None:
         do_script(shlex.quote(args.script_exec), use_exec=True)
 
-    if args.net:
+    if args.bridge:
+        qemuargs.extend(['-netdev', 'tap,helper=/usr/lib/qemu/qemu-bridge-helper,id=hn0', '-device', 'virtio-net-pci,netdev=hn0,id=nic1,mac='+random_mac()])
```

Comment:
What does 'hn0' mean?

There should be either usable semantics or an error if --net and --bridge are both specified.

We should error out if /usr/lib/qemu/qemu-bridge-helper isn't available at that at that path.  And maybe we should search more than one place.